src/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -387,6 +387,23 @@ public void TestStartWithNonExistingUserThrows()
             Assert.Throws<Win32Exception>(() => p.Start());
         }
 
+        [Fact]
+        public void TestExitCodeKilledChild()
+        {
+            using (Process p = CreateProcessLong())
+            {
+                p.Start();
+                p.Kill();
+                p.WaitForExit();
+
+                Assert.True(p.ExitCode > 128, "Exit code should be 128 + SIGKILL.");
```

Comment:
nit, you could remove this assert now, but it's harmless...

Reply:
If we keep it, it'd be better to use InRange so that we get the actual value in the assert, e.g.
```C#
Assert.InRange(p.ExitCode, 129, int.MaxValue); // Exit code should be 128 + SIGKILL
```